subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/store.py

```diff
@@ -777,5 +779,108 @@ def parse_data():
     # delegate the parsing to the simple_store
     store.register_custom_rewrite_settings(_rewrite_settings)
     store_simple.register_custom_parse_results(_parse_directory)
+    store_simple.register_custom_lts_parse_results(_parse_lts_dir)
 
     return store_simple.parse_data()
+
+def _parse_lts_dir(add_to_matrix, dirname, import_settings):
+    with open(f'{dirname}/data.json') as f:
+        data = json.load(f)
+        add_to_matrix(_encode_json(data, dirname, data['settings']))
+
```

Comment:
quick Python formatting rule: when defining things at the top level (beginning of the line), they should be separated by two empty lines. When it's inside eg a class, only one line

see https://stackoverflow.com/questions/2953250/python-pep8-blank-lines-convention for examples and references